Applications/Games/Space Engineers/Steam/script.js

```
@@ -9,8 +9,8 @@ new SteamScript()
     .name("Space Engineers")
     .editor("Keen Software House")
     .author("Zemogiter")
-    .appId(244850)
-    .wineVersion("4.11")
+    .appId("244850")
```

@plata should the `appId` be passed as a string or as a number?

---

Does it matter?

---

I don't think so but I think we should do it in a uniform manner.

---

Isn't it uniform in the other Steam scripts?

---

According to other scripts `appId` should be a number, see:

https://github.com/PhoenicisOrg/scripts/blob/920c415c38cfffca58d3813108893d1c50c69f44/Applications/Games/Enderal/Steam/script.js#L8

https://github.com/PhoenicisOrg/scripts/blob/920c415c38cfffca58d3813108893d1c50c69f44/Applications/Games/BRINK/Steam/script.js#L7

https://github.com/PhoenicisOrg/scripts/blob/920c415c38cfffca58d3813108893d1c50c69f44/Applications/Games/BioShock/Steam/script.js#L7

Maybe we should enforce this and throw an error if some invalid input has been given?

---

Well I have to renistall the prefix anyways because Wine stuck with ALSA again and any attempt at changing it to pulse results in what I have written in OP. Might as well check if passing `appId` as a string matters.

---

> Maybe we should enforce this and throw an error if some invalid input has been given?

Yes.

---

This leaves us with the initial question:

> Should the `appId` be passed as a string or as a number?

---

When installing `Space Engineers` I get the same error from https://github.com/PhoenicisOrg/scripts/issues/1098#issuecomment-525303874

---

It should be number.

---

Changing `appId` back to number changes nothing. Still getting that error.

---

@Zemogiter I think you misunderstand the comment discussions. This is not about the error you are getting. We simply want to have a uniform style how the `appId` should be passed to the `SteamScript`.

---

@madoar understood